service/src/test/java/de/starwit/mapper/ApplicationMapperTest.java

```diff
@@ -0,0 +1,127 @@
+package de.starwit.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import de.starwit.dto.ApplicationDto;
+import de.starwit.dto.EntityDto;
+import de.starwit.dto.FieldDto;
+import de.starwit.dto.FieldType;
+import de.starwit.persistence.entity.App;
+import de.starwit.persistence.entity.AppTemplate;
+import de.starwit.persistence.entity.Attribute;
+import de.starwit.persistence.entity.DataType;
+import de.starwit.persistence.entity.Domain;
+
+@RunWith(SpringRunner.class)
+public class ApplicationMapperTest {
+
+    final static Logger LOG = LoggerFactory.getLogger(ApplicationMapperTest.class);
+
+        /**
+     * <pre>
+     * To check the Service class, we need to have an instance of Service class created and available as a
+     * &#64;Bean so that we can @Autowire it in our test class.
+     * This configuration is achieved by using the @TestConfiguration annotation.
+     * </pre>
+     */
+    @TestConfiguration
+     static class ApplicationMapperTestConfiguration {
+
+        @Bean
+        public ApplicationMapper createApplicationMapper() {
+            ApplicationMapper applicationMapper = new ApplicationMapper();
+            return applicationMapper;
+        }
+
+        @Bean
+        public EntityMapper createEntityMapper() {
+            EntityMapper entityMapper = new EntityMapper();
+            return entityMapper;
+        }
+
+        @Bean
+        public FieldMapper createFieldMapper() {
+            FieldMapper fieldMapper = new FieldMapper();
+            return fieldMapper;
+        }
+    }
+
+    private App app;
+    private ApplicationDto dto;
+
+    @Autowired
+    private ApplicationMapper applicationMapper;
+
+    @Before
+    public void createApp() {
```

Comment:
Maybe this could be outsourced in a separate file for a 'Entity/Domain collection'